feature/visited/src/main/kotlin/uk/govuk/app/visited/ui/EditVisitedScreen.kt

```diff
@@ -54,45 +54,51 @@ internal fun EditVisitedRoute(
 
     EditVisitedScreen(
         uiState = uiState,
-        onRemove = {
-            viewModel.onRemoveClick()
-            viewModel.onRemove()
-            onBack()
-        },
-        onEditPageView = { viewModel.onEditPageView() },
-        onBack = {
-            viewModel.onDoneClick()
-            onBack()
-        },
-        onSelect = { title, url ->
-            viewModel.onSelect(title, url)
-        },
-        onSelectAll = {
-            viewModel.onSelectAllClick()
-            viewModel.onSelectAll()
-        },
-        onDeselectAll = {
-            viewModel.onDeselectAllClick()
-            viewModel.onDeselectAll()
-        },
+        actions = EditVisitedScreenActions(
+            onEditPageView = { viewModel.onEditPageView() },
+            onBack = {
+                viewModel.onDoneClick()
+                onBack()
+            },
+            onSelect = { title, url ->
+                viewModel.onSelect(title, url)
+            },
+            onSelectAll = {
+                viewModel.onSelectAllClick()
+                viewModel.onSelectAll()
+            },
+            onDeselectAll = {
+                viewModel.onDeselectAllClick()
+                viewModel.onDeselectAll()
+            },
+            onRemove = {
+                viewModel.onRemoveClick()
+                viewModel.onRemove()
+                onBack()
+            }
+        ),
         modifier = modifier
     )
 }
 
+private class EditVisitedScreenActions(
```

Comment:
This looks like it could be a data class?

Reply:
Not much point I don't think as it doesn't hold any actual data. You're not going to want to do equality checks or copy it etc as it is just a collection of lamdbas.